journal.lic

```diff
@@ -10,17 +10,23 @@ class WriteJournal
   def initialize
     settings = get_settings
     @journal_noun = settings.journal_noun
+    UserVars.journal_last_use ||= Time.now - 86400
 
     write_journal
   end
 
   def write_journal
+    return if Time.now - UserVars.journal_last_use < 86400
+
     case bput("remove my #{@journal_noun}", 'You remove', 'You detach', 'What were', 'Remove what')
     when 'Remove what', 'What were'
       echo 'No journal, exiting.'
       exit
     end
-    bput("write my #{@journal_noun}", 'You write', 'Flipping to a blank page', 'Having recently written', 'As you open your', "You can't do that")
+    case bput("write my #{@journal_noun}", 'You write', 'Flipping to a blank page', 'Having recently written', 'As you open your', "You can't do that")
+    when 'You write'
```

Comment:
did it this way just in case for some reason your journal isn't available you wont reset the timer